### ISSUE TITLE
[deconz] Support thing type ZHACarbonMonoxide (e.g. Heiman CO sensor)

### ISSUE THREAD

**Issue by derdon999** [openhab/openhab-addons#7643]
**Environment**
- Version used: openHAB 2.5.4 release, deCONZ binding 2.5
- Raspbian GNU/Linux 10 (buster)
- ConBee II

According to deCONZ Plugin Help all sensors are supported, including smoke/fire/CO sensors.
The sensor is also listed by the REST query:

> 
    "30": {
      "config": {
        "battery": 100,
        "on": true,
        "reachable": true
      },
      "ep": 1,
      "etag": "1902d2cdb9d256624398f9ec3d35481d",
      "manufacturername": "Heiman",
      "modelid": "COSensor-N",
      "name": "CO Sensor",
      "state": {
        "carbonmonoxide": null,
        "lastupdated": "none",
        "lowbattery": null,
        "tampered": null
      },
      "swversion": "2018.4.22",
      "type": "ZHACarbonMonoxide",
      "uniqueid": "00:28:9f:00:03:eb:80:2a-01-0500"
> 

Is it possible to integrate it into the binding in the near future?

Many thanks,

Raphael

**Comment by J-N-K** [comment]
It's easy to integrate, but can you show the REST response when a value is present?

**Comment by derdon999** [comment]
Sorry, I´m new to RESTclient.
If you mean METHOD: options and URL: http://192.168.178.77:8090/api/9XXXXXXXXB/30
then this is the response:

> 

Status Code: 200 OK
access-control-allow-credentials: true
access-control-allow-headers: Authorization, Access-Control-Allow-Origin, Content-Type
access-control-allow-methods: POST, GET, OPTIONS, PUT, DELETE
access-control-allow-origin: *
access-control-expose-headers: Gateway-Name, Gateway-Uuid
access-control-max-age: 0
cache-control: no-store, no-cache, must-revalidate, post-check=0, pre-check=0
connection: close
content-length: 0
content-type: text/html
gateway-name: openhab-coinbee2
gateway-uuid: 40c1a501-6e3f-48a6-9a33-4005g6ab02ch
pragma: no-cache

>

**Comment by J-N-K** [comment]
The snippet before was good, I just need to see what `carbonmonoxide` can show if it is not `null` (e.g. a number, `true`, `false`)

**Comment by derdon999** [comment]
Hi,
unfortunately I was not able to make any change to the status of carbonmonoxide `null` or lastupdated as seen in the first post. 
I tried pressing the test button on the sensor, reconnected to deconz again (response: new sensor found but not listed in the list of sensors) and triggered the alarm with some smoke. The alarm on the sensor turned on, the led was blinking in yellow but no change in status of carbonmonoxide as I can see in the RESTclient nor any trigger changed in openHAB (paper UI, control channel) after I manually added the CO-sensor as a deconz device with its uniqueid (sensor is showing online for quite some time).
